src/app/providers/provider-form/gitlab/gitlab.component.html

```diff
@@ -3,11 +3,13 @@
     <h5><label>URL <span class="required">*</span></label></h5>
     <span *ngIf="providerForm.get('url').validator"></span>
     <input pInputText type="text" placeholder="URL" formControlName="url" class="form-input">
+    <p-tag *ngIf="!providerForm.controls.url.valid && providerForm.controls.url.enabled" severity="danger" value="The url '{{providerForm.controls.url.value}}' is not valid!"></p-tag>
```

Comment:
I think including the URL value here in the tag is excessive. The value is already shown in the field just above the tag.

```suggestion
    <p-tag *ngIf="!providerForm.controls.url.valid && providerForm.controls.url.enabled" severity="danger" value="The url is not valid!"></p-tag>
```